fvm_actor_utils/src/messaging.rs

```diff
@@ -52,8 +52,8 @@ pub trait Messaging {
     fn initialize_account(&self, address: &Address) -> Result<ActorID>;
 }
 
-// the method number comes from taking the name as "Received" and applying
-// the transformation described in https://github.com/filecoin-project/FIPs/pull/399
+// the method number comes from taking the name as "Receive" and applying
+// the transformation described in [FRC-0042](https://github.com/filecoin-project/FIPs/blob/master/FRCs/frc-0042.md)
```

Comment:
Should these be doc comments if using markdown syntax?

Reply:
Done in followup https://github.com/helix-onchain/filecoin/pull/108